src/averageAges.js

```diff
@@ -14,13 +14,26 @@
  *
  * @return {number}
  */
+
+// Shared Functions
+const age = (timePeriod) =>
+  timePeriod.map(person =>
```

Comment:
No need to use `map` method here. You can do it inside your `reduce` callback function.